reframe/frontend/executors/policies.py

```diff
@@ -205,7 +209,11 @@ def on_task_failure(self, task):
 
     def on_task_success(self, task):
         msg = f'{task.info()}'
-        self.printer.status('OK', msg, just='right')
+        if task.check.dry_run_mode:
+            self.printer.status('DOK', msg, just='right')
+        else:
+            self.printer.status('OK', msg, just='right')
+
```

Comment:
I'd rather keep the original behaviour and colors everywhere, except for changing `RUN` to `DRY`.

Reply:
I thought the color were useful when we suggested people to run with `-Sdry_run_mode=1`, especially for tests with fixtures.
Even if you see a lot of `DRY` messages it can be confusing in the end if the `OK` belongs to a dry-run or a normal run. But if I understand correctly the user will have to run everything in dry-run, we don't allow them to choose if they want to run something normally, right?

Reply:
Exactly, I would vote for everything in dry-run. Technically, depending on where the `--dry-run` option sets the corresponding variable, we could allow exceptionally to reset the dry run mode for a specific test. But I would start simple. And then `DOK` instead of `OK` is a bit weird as a name, isn't it?

Reply:
> And then DOK instead of OK is a bit weird as a name, isn't it?

Yes, I was removing the `D` from the message when it was printed, I just used it to differentiate from normal `OK` and set a different color. But if we are running all the tests in dry-run, `OK` is enough, I will remove it.